### ISSUE TITLE
missing_docs does not trigger for a class inherited from a documented class. 

### ISSUE THREAD

**Issue by rabbitinspace** [realm/SwiftLint#3328]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

I noticed that if there's a public class that is inherited from a documented parent class then `missing_docs` rule does not trigger if the child class doesn't have a documentation. Here's an example:

```swift
/// AAAA
public class AAAA {
    /// aaaa
    public func aaaa() {}
}

// not triggered
public class BBBB: AAAA {
    // not triggered
    public func bbbb() {}
}
```

I expect `missing_docs` rule to trigger for the `BBBB` class and for its `bbbb` method but it doesn't trigger. 

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
Loading configuration from '.swiftlint.yml'
Linting Swift files at paths
Linting 'ViewController.swift' (1/1)
Done linting! Found 0 violations, 0 serious in 1 file.
```

### Environment

* SwiftLint version:
 **0.40.1**

* Installation method:
 **Homebrew**

* Paste your configuration file:
```yml
opt_in_rules:
  - missing_docs

```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
**No**

* Which Xcode version are you using (check `xcodebuild -version`)?
**Xcode 11.6
Build version 11E708**

* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.
```swift
/// AAAA
public class AAAA {
    /// aaaa
    public func aaaa() {}
}

// missing_docs not triggered here
public class BBBB: AAAA {
    // missing_docs not triggered here
    public func bbbb() {}
}
```

**Comment by SimonMcFarlane** [comment]
The [docs](https://realm.github.io/SwiftLint/missing_docs.html) specify that the configuration setting `excludes_extensions` has a default of `true`. Setting this to `false` should do the job.

```
missing_docs:
  excludes_extensions: false
  excludes_inherited_types: false
  excludes_trivial_init: true
  error:
   - fileprivate
   - internal
   - open
   - private
   - public
```
